src/main/java/teammates/ui/webapi/action/GetInstructorPrivilegeAction.java

```diff
@@ -1,14 +1,84 @@
 package teammates.ui.webapi.action;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.http.HttpStatus;
+
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 import teammates.common.exception.UnauthorizedAccessException;
 import teammates.common.util.Const;
-import teammates.ui.webapi.output.ApiOutput;
+import teammates.ui.webapi.output.InstructorPrivilegeData;
 
 /**
  * Get the instructor privilege.
  */
 public class GetInstructorPrivilegeAction extends Action {
+    static final Map<String, InstructorPrivilegeData> INSTRUCTOR_PRIVILEGES = new HashMap<>();
```

Comment:
This looks like it should be set in `Const` class instead.

Reply:
It will cause this illegal dependency that cannot pass static analysis

Reply:
Just made aware of it. Then it should be ok for now.